perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_userPreferredCertDN.java

```diff
@@ -40,7 +40,7 @@ public void checkAttributeValue(PerunSessionImpl sess, User user, Attribute attr
 			else return;
 		}
 		if(attribute.getValue() == null) {
-			if(certDNsValue != null || !certDNsValue.isEmpty()) throw new WrongAttributeValueException(attribute, user, "This attribute value can't be null because of notNull attribute userCertDNs");
```

Comment:
I think there should be `&&` instead of `||` and the condition should not be removed.